src/layer-manager/lib/generators/heatmap/heatmap.js

```diff
@@ -0,0 +1,260 @@
+import memoizeOne from 'memoize-one'
+import paintByGeomType from './heatmap-layers-paint'
+
+export const HEATMAP_TYPE = 'HEATMAP'
+
+const FAST_TILES_KEY = '__fast_tiles__'
+const DEFAULT_FAST_TILES_API = 'https://fst-tiles-jzzp2ui3wq-uc.a.run.app/v1/'
+const BASE_WORKER_URL = `http://${FAST_TILES_KEY}/{z}/{x}/{y}`
+
+export const GEOM_TYPES = {
+  BLOB: 'blob',
+  GRIDDED: 'gridded',
+  EXTRUDED: 'extruded',
+}
+
+export const GEOM_TYPES_GL_TYPES = {
+  [GEOM_TYPES.BLOB]: 'heatmap',
+  [GEOM_TYPES.GRIDDED]: 'fill',
+  [GEOM_TYPES.EXTRUDED]: 'fill-extrusion',
+}
+
+export const COLOR_RAMPS = {
+  FISHING: 'fishing',
+  PRESENCE: 'presence',
+  RECEPTION: 'reception',
+}
+
+const COLOR_RAMPS_RAMPS = {
+  [COLOR_RAMPS.FISHING]: [
```

Comment:
is this a fixed ramp color, or will have different colors depending on the product where it will be used? 
because we could expose the configuration in that case

Reply:
I am not sure yet and this will probably be changed, but for now I went with a conservative option.
For the moment it seems we'll be using defined ramps to stick with the GFW brand/style guide, and we'll probably add more options, ie `BIVARIATE` / `DIVERGENT` /  `CATEGORICAL` etc